packages/react-navigation/src/rum/instrumentation/DdRumReactNavigationTracking.tsx

```diff
@@ -30,14 +30,22 @@ export class DdRumReactNavigationTracking {
 
     private static navigationStateChangeListener: NavigationListener;
 
-    private static appStateListener: AppStateListener;
-
     private static viewNamePredicate: ViewNamePredicate;
 
+    private static backHandler: NativeEventSubscription | null;
+
     static ROUTE_UNDEFINED_NAVIGATION_WARNING_MESSAGE = "A navigation change was detected but the RUM ViewEvent was dropped as the route was undefined."
     static NULL_NAVIGATION_REF_ERROR_MESSAGE = "Cannot track views with a null navigationRef."
     static NAVIGATION_REF_IN_USE_ERROR_MESSAGE = "Cannot track new navigation container while another one is still tracked. Please call `DdRumReactNavigationTracking.stopTrackingViews` on the previous container reference."
 
+
+    static onBackPress = () => {
+        if (DdRumReactNavigationTracking.registeredContainer === null) return false;
+        if (DdRumReactNavigationTracking.registeredContainer.canGoBack()) return false;
+        DdRumReactNavigationTracking.stopTrackingViews(DdRumReactNavigationTracking.registeredContainer);
+        return false;
```

Comment:
shouldn't it be `true` here? Otherwise this function always return `false`, meaning we don't need actually return value.

Reply:
In our case, we always want to return `false`. By the documentation (https://reactnative.dev/docs/backhandler), if we return true, earlier callbacks won't be called (such as the React-Navigation callback). In the last case, if we return true the app won't exit at all when we press back.

I can extract the logic that stops the tracking in a separate function so we only write `return false` once (and I can point to the docs to explain why we return false), is that ok for you?

Reply:
Thanks for the explanation, this is fine as it is then, no changes needed here - documentation of `BackHandler` is explaining why we always return `false` here.

Reply:
It is actually like in Android when you intercept an event. If you return `true` means you are consuming the event and it will not be propagated to the upper listeners.